funnel/templates/session_view_popup.html.jinja2

```diff
@@ -27,13 +27,13 @@
 </div>
 <div class="modal__body">
   <div class="section">
-    {% if session.description_html %}
+    {% if session.description %}
       <h3 class="mui--text-title"><strong>{% trans %}Description{% endtrans %}</strong></h3>
-      <div class="mui--text-body2 image-wrapper">{{ session.description_html|safe }}</div>
+      <div class="mui--text-body2 image-wrapper">{{ session.description.html }}</div>
```

Comment:
Just use `session.description` in Jinja2. It will automatically use the HTML version.